### DISCUSSION TITLE
Delete deployment made from unknown account

### DISCUSSION THREAD

**Discussion by arunlodhi** [vercel/community#3537] [Help] [answered]
### Summary

My github org was linked to vercel and a deployment was created where my old website has been deployed. However, I don't see this deployment in my vercel account. I would need help in knowing which account this deployment is related to and  how can I remove it. The deployment url is:
https://front-space-mr.vercel.app/

### Example

https://front-space-mr.vercel.app/

### Steps to Reproduce

N/A

**Comment by amyegan** [comment, marked as answer]
Hey @arunlodhi. If you believe the deployment contains malicious or illegal content, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team or email [abuse@vercel.com](mailto:abuse@vercel.com) where the request will be dealt with immediately. Please provide as much relevant information as possible, taking into consideration our [DMCA policy](https://vercel.com/legal/dmca-policy) and [trademark policy](https://vercel.com/legal/trademark-policy).

**Reply by arunlodhi** [reply]
Thanks!

We were able to track the deployment to a dormant account belonging to an ex-teammate and have deleted the related project.
